pcp/Platform.c

```diff
@@ -370,6 +370,10 @@ void Metric_enableThreads(void) {
 }
 
 bool Metric_fetch(struct timeval *timestamp) {
+   if (pcp->result) {
+      pmFreeResult(pcp->result);
```

Comment:
If `pmFreeResult` is `NULL`-safe, why check for `NULL` here? If it's not, then `Platform_done` should have that check too.

Reply:
Yep, needs a NULL check in done too - thanks!